src/index.html

```diff
@@ -11,7 +11,167 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class ="fieldset">
+        <legend>Personal information</legend>
+        <label class="form__field">
+          Surname:
+          <input
+          type="name"
+          name="surname"
+          class="field"
+          pattern="\w+"
+          required
+          autocomplete="off"
+          >
+        </label>
+        <label class="form__field">
+          Name:
+          <input
+          type="name"
+          name="name"
+          class="field"
+          pattern="\w+"
+          required
+          autocomplete="off"
+          >
```

Comment:
```suggestion
          <input
            type="name"
            name="name"
            class="field"
            pattern="\w+"
            required
            autocomplete="off"
          >
```

And correct the same over your code.